pyctbgui/utils/plotPattern.py

```diff
@@ -84,8 +84,8 @@ def patternPlot(self):
 
         # Load the pattern and get all lines
         # Loop all lines
-        #with open(Folder + "/" + File_pat + ".pat") as f_pat:
-        with Path.open(self.pattern) as f_pat:
+        # with open(Folder + "/" + File_pat + ".pat") as f_pat:
+        with Path(self.pattern).open() as f_pat:
             lines_pat = f_pat.readlines()
         f_pat.close()
```

Comment:
Strictly not a part of the PR since the line didn't change but the close is redundant. Exiting the with block closes the file.